### PR TITLE
2.0 Milestone 🎨

### PR DESCRIPTION
**Pull request by lluia**
## Roadmap &nbsp; 🗺

Summarising progress of our `2.0` milestone here 💅🏻

### Rebrand 
- [x] #51 **New typography** 
  - [x] documentation on how to apply them
- [x] #52 **New colors**
  - [x] documentation on how to apply them

### Cleanup
- [ ] #49 ~~**Rename to `<Label />`**~~ ( moved to **`3.0`** )
- [x] #30 **Remove social media icons**
- [x] #20 **Better naming for icons**
- [x] #21 **Remove `<Card type="button" />`**
- [ ] #17 ~~**Standardise `<Spinner />`**~~ ( moved to **`3.0`** )
  - [ ] ~~catch-up with design team about it~~

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# React components
+# React components `2.0`
```
Comment (lluia) [inline README.md]: Lets block merge of this for now 😄

File: src/constants/colors.ts [hunk 2]
```diff
@@ -1,139 +1,21 @@
...
+    white: '#FFFFFF',
+    light: '#EAEAEE',
+    medium: '#C5C6CD',
+    dark: '#0D0A38',
+  },
+  semantic: {
+    success: '#1EC06A',
+    alert: '#FFB428',
+    error: '#EE0505',
+  },
```
Comment (alexmngn) [inline src/constants/colors.ts]: Can we match those naming with the one in the brand document?

https://marvelapp.com/project/4236000/screen/60617108

Comment (alexmngn) [reply src/constants/colors.ts]:
```
colors.tealPrimary
colors.tealSecondary
colors.bluePrimary
colors.blueSecondary
colors.surfaceDark
colors.surfaceBackground
```

Comment (thegrinder) [reply src/constants/colors.ts]: I don't think we should follow the design spec to the letter just yet as it's not fully fleshed out. There is plenty of overlap and we'd be running into issues, e.g.:
-  is `#0D0A38` `surfaceDark` or `textPrimary`, or `onSurfaceDark`?
- is `#FFFFFF`  `onSecondary`, `onSurfaceDark`, or just `White`?
etc.

Comment (lluia) [reply src/constants/colors.ts]: I think at the end of the day the important is that we have a restricted set of colors to use and clear guidelones on how to apply them 🎨 &nbsp;💅🏻

The design team has seen our colours spec and hasn't complain that the name should match theirs 😌

I would wait until the *new design system* matures... it's still plenty to do and spec and the new guidelines hasn't even been applied on any web product yet.

**Pull request merged**